modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with condition type, informing you of potential impacts to upgrading your cluster. 
+The following condition types have consistent wording across all ClusterVersion and ClusterOperator conditions. Individual components may also have additional condition types with component-specific language.
```

Comment:
nit: I'm not clear on "consistent wording", but what this sentence is trying to say is that when you see an `Available=False` condition in a ClusterVersion or ClusterOperator, you can use the `Available` definition in these docs (or in [the Go API][1]) to understand the high-level meaning (e.g. "it's bad; send an admin") without having to bother to look at the condition's `reason` or `message`.  Maybe one of the following:

> The following condition types have consistent semantics across...
> The following condition types are consistent across...
> The following condition types have the same meaning across...

or something like that?

[1]: https://github.com/openshift/api/blob/cce310ad2932f6de24491052d506926e484c082c/config/v1/types_cluster_operator.go#L146-L150